redisson/src/main/java/org/redisson/client/handler/CommandPubSubDecoder.java

```diff
@@ -255,8 +251,7 @@ protected Decoder<Object> selectDecoder(CommandData<Object, Object> data, List<O
             if (parts.size() == 2 && "pmessage".equals(parts.get(0))) {
                 return ByteArrayCodec.INSTANCE.getValueDecoder();
             }
-            
-            if (parts.size() == 2 && "message".equals(parts.get(0))) {
+            if (parts.size() == 2 && TYPE_MESSAGES.contains(parts.get(0).toString())) {
```

Comment:
TYPE_MESSAGES has "pmessage" but it can't be handled using this code

Reply:
yes, that's right this case is only for "message" and "smessage" because they both have parts.size == 2, when "pmessage" has parts.size == 3